framework/test/mocha/integration/blocks/verify/verify.js

```diff
@@ -423,11 +423,15 @@ describe('blocks/verify', () => {
 				}
 			});
 
-			it('should fail when block generator is invalid (fork:3)', async () => {
+			it('should fail when block generator is invalid', async () => {
 				try {
 					await library.modules.processor.process(block2);
 				} catch (err) {
-					expect(err.message).equal('Failed to verify slot: 3377288');
+					expect(err.message).equal(
+						`Failed to verify slot: 3377288. Block ID: ${
```

Comment:
Would you like to decouple this error message by using `${block2.timestamp}` instead of `3377288`? 🥇

Reply:
Slot and timestamp aren't the same think and there is no slot data in blocks

Reply:
Ahhh I thought it was the timestamp from the fixed block, OK!